tests/integration/hive_metastore/test_workflows.py

```diff
@@ -1,11 +1,11 @@
 from datetime import timedelta
 
 import pytest
-from databricks.sdk.errors import NotFound
+from databricks.sdk.errors import InvalidParameterValue, NotFound
 from databricks.sdk.retries import retried
 
 
-@retried(on=[NotFound], timeout=timedelta(minutes=5))
+@retried(on=[NotFound, InvalidParameterValue], timeout=timedelta(minutes=5))
```

Comment:
It gets an `InvalidParameterValue` when requesting the job run id. It confuses me a bit, as we delete jobs, but the runs should stay

Reply:
Jobs and clusters apis return 400 instead of 404, it's confusing